lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -61,9 +61,9 @@ class UsesHTTP2Audit extends Audit {
       let displayValue = '';
       if (resources.length > 1) {
         displayValue =
-          `${Util.formatNumber(resources.length)} requests were not handled over HTTP/2`;
+          `${Util.formatNumber(resources.length)} requests not served via HTTP/2`;
```

Comment:
keep the 'were'?

Reply:
I don't care much, but it is a little weird since the audit touched above keeps the verb :)